pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -513,4 +513,7 @@ message LogicalReplicationWriterSpec {
 
     // TableDescriptors is a map between from destination table IDs to the source table descriptor.
     map<int32, cockroach.sql.sqlbase.TableDescriptor> table_descriptors = 8 [(gogoproto.nullable) = false];
+
+    // TablePrefixes is a map between from destination table IDs to its database and schema prefixes.
+    map<int32, string> table_prefixes = 9 [(gogoproto.nullable) = false];
```

Comment:
How about, rather than storing the pre-built prefix, we either store the entire descriptor or we store a new message type that has the database name and schema name as fields.  Then we wouldn't have to re-parse out the database name I think this would help with some other work we are doing as well.

Reply:
works for me!